platform/podjs/src/browserPod.ts

```diff
@@ -671,15 +673,14 @@ export class BrowserPod implements Pod {
 
     constructor() {
         window.addEventListener("load", async () => {
-            if (!window.manifestData) {
+            if (MANIFEST_DATA === {}) {
```

Comment:
I guess this change is unrelated, right?

Reply:
it is the change indeed... we now replace the `window.manifestData` at the top of the file with the provided script, so MANIFEST_DATA has the json data at this ^ point

Reply:
Sorry, I should have made myself clearer here. `MANIFEST_DATA` is now a constant that holds `windows.manifestData`; that's good refactoring here. But the direct replacement here would have been `!MANIFEST_DATA`. You're now comparing it to `{}`; that's the unrelated change I'm talking about. It's probably the same functionality, and in my book, the initial mode was shorter (and a bit more general), that's why I inquired about it.